docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -35,7 +35,7 @@ The main logic of the game is in the following `create_game` function.
 /// `x_address` and `o_address` are the account address of the two players.
 public entry fun create_game(x_address: address, o_address: address, ctx: &mut TxContext) {
     // TODO: Validate sender address, only GameAdmin can create games.
-    
+
     let id = object::new(ctx);
     let game_id = object::uid_to_inner(&id);
```

Comment:
This should be switched to a receiver function, right?

Reply:
`id.to_inner()` seems weird to me, and since we never concluded bikeshedding it thought it best to leave it as-is for now